testing/pysubnettree/search-all.test

```diff
@@ -0,0 +1,46 @@
+# Test that search all returns the correct entries.
+#
+# @TEST-EXEC: python3 %INPUT
+
+from testsetup import testcase, ipv4binary, ipv6binary
+import SubnetTree
+
+def ipv4_func(ipstr):
+    return ipstr
+
+def ipv6_func(ipstr):
+    return ipstr
+
+def do_tests(binarymode):
+    if binarymode:
+        v4 = ipv4binary
+        v6 = ipv6binary
+    else:
+        v4 = ipv4_func
+        v6 = ipv6_func
```

Comment:
This test doesn't use `v4` or `v6`, so this is causing confusion, but see comment about implementing binarymode in which case you might want to use them.